test/pkg/containerd/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# alpine 3.5 lacks go1.8
+FROM golang:1.8
+RUN apt update && apt install -y btrfs-tools
+ENV CONTAINERD_COMMIT=199544ea8009bf6988a81456c7dec66b2d3268e0
```

Comment:
I was thinking of moving the `containerd` git checkout into the `linuxkit/alpine` package then we could use the same one.